python/tests/stress/arcticdb/version_store/test_stress_generated.py

```diff
@@ -82,3 +91,22 @@ def test_stress_small_row(pct_null, repeats, symbol, lmdb_version_store_tiny_seg
     elapsed = time.time() - start
     print("arctic read time: " + str(elapsed))
     assert_frame_equal(test_df, df)
+
+
+@pytest.mark.parametrize("ncols, nrows", [(5, 10000), (50, 100000), (50, 1000000), (50, 10000000), (500, 1000000)])
```

Comment:
Minor: if you intend to merge this, it's not worth running the smaller test cases unless there are any stats being collected automatically.

Reply:
I will not merge this part of the code. It will be removed after the benchmark.